edweiss-app/components/notifs/NotifDisplay.tsx

```diff
@@ -38,13 +38,7 @@ const NotifDisplay: ReactComponent<{ item: NotifList.Notif, id: string, dateSect
     // Render left actions on swipe
     const renderLeftActions = () => (
         <TTouchableOpacity onPress={() => {
-            if (item.read) {
-                markAsUnreadAction();
-            } else {
-                markAsReadAction();
-                // const date: Timestamp = Timestamp.fromDate(new Date(1730697600000));
-                // pushNotifAction('meeting', 'Meeting', 'Test Add Notif', false, null, OurTime.toDate(date).toISOString());
-            }
+            if (item.read) markAsUnreadAction(); else markAsReadAction();
```

Comment:
Same as above, a core logic like this shouldn't be inlined with the rest of the rendering as it would be a nightmare to maintain and refactor it, this should be in its own function above (with [`useCallback`](https://react.dev/reference/react/useCallback) preferably !).